<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaintProgram</class>
 <widget class="QWidget" name="PaintProgram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PaintProgram</string>
  </property>
  <widget class="QPushButton" name="setBrushButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>60</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Brush</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>30</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Brush Width</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="numberNights">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>30</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="setSquareButton">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>60</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Square</string>
   </property>
  </widget>
  <widget class="QLabel" name="canvasLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>711</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
